cupy/linalg/decomposition.py

```diff
@@ -290,12 +309,14 @@ def svd(a, full_matrices=True, compute_uv=True):
         raise linalg.LinAlgError(
             'Parameter error (maybe caused by a bug in cupy.linalg?)')
 
-    # Note that the returned array may need to be transporsed
+    # Note that the returned array may need to be transprosed
```

Comment:
A nitpick but a typo.
```suggestion
    # Note that the returned array may need to be transposed
```